core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -326,13 +326,22 @@ public BackupResponse backupRemotely(Collection<XSiteBackup> backups, XSiteRepli
          DeliverOrder order = xsb.isSync() ? DeliverOrder.NONE : DeliverOrder.PER_SENDER;
          long timeout = xsb.getTimeout();
          try {
-            sendCommand(recipient, command, request.getRequestId(), order, rsvp, false, false);
+            Message message = new Message(toJGroupsAddress(recipient));
+            marshallRequest(message, command, request.getRequestId());
+            setMessageFlags(message, order, rsvp, false);
+
+            remoteExecutor.submit(() -> {
```

Comment:
-1
sending order is important. You need to use a single thread to send the request or you risk it to be reordered.

Reply:
Oops :)

Reply:
I added `ActionSequencer` when the commands need to be ordered